### PR TITLE
Memory

### PR DESCRIPTION
**Pull request by mnsinger**
Works with basic styling and functionality.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,130 @@
...
   </head>
   <body>
+  <div id="board"></div>
+
+<script>
+
+
+// GLOBAL VARIABLES TO REMEMBER THE FIRST CARD FLIP
+var c = 0;
+var firstval;
```
Comment (afeld) [inline index.html:22]: How could you avoid making these global?

File: index.html [hunk 2]
```diff
@@ -2,7 +2,130 @@
...
+	var rowString = "";
+	// CREATES BOARD OF ROWS x COLS
+	for (var i=0; i<rows; i++) {
+		rowString="<tr>";
+		for (var j=0; j<cols; j++) {
+			rowString+="<td>.</td>";
+		}
+		rowString+="</tr>";
+		$('#board').children().append(rowString);
+	}
```
Comment (afeld) [inline index.html:40]: Nice! I would move the board creation into its own method, so that it's clear what the code is doing without even needing a comment.

File: index.html [hunk 3]
```diff
@@ -2,7 +2,130 @@
...
+						} 
+					}
+					
+				}
+				// ON SECOND FLIP ALWAYS SET C TO ZERO TO RESET
+				c=0;
+			}; 
+		    console.log(this.innerHTML);
+		}
+	}
```
Comment (afeld) [inline index.html:120]: This method is huge...how could you make it easier to follow?

File: index.html [hunk 4]
```diff
@@ -2,7 +2,130 @@
...
+		$('#board').children().append(rowString);
+	}
+}
+
+Board.prototype.setBoard = function() {
+
+	var table = document.getElementById('tid');
+	var cells = table.getElementsByTagName('td');
+	var values = new Array();
+	var pairValues = new Array();
```
Comment (afeld) [inline index.html:48]: Minor: I would use array literals here instead (`[]`), unless you have a specific reason to need the constructor form.

**Comment by afeld** [comment]
Well done! Your inline comments are good, but I would think about how to make your code more self-documenting. The main way to do this is to break it into smaller chunks.

The [build output](https://travis-ci.org/advanced-js/memory/builds/87578516) is hard to follow because it's giving you a bunch of warnings about the included jQuery file (working on fixing that), but try pasting your JS into http://jshint.com/ and fixing the warnings.

File: index.html (hunk 3 above)
Comment (mnsinger) [inline index.html:120]: Gotcha. I'm guessing this should be broken up and given names that could make the code easier to follow.

Comment (afeld) [inline index.html:120]: Sounds good to me :wink:
